backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/GwtGraphics.java

```diff
@@ -576,6 +576,10 @@ public void setVSync (boolean vsync) {
 	public void setForegroundFPS (int fps) {
 	}
 
+	@Override
+	public void setBackgroundFPS (int fps) {
+	}
```

Comment:
Not sure why this was added here but not in `AndroidGraphics` and `IOSGraphics` (which also have empty overrides). I don't think it's important but just don't see reason for GWT to get special treatment.